### PR TITLE
[SDK-460] Remove tracking_type from create budget

### PR DESCRIPTION
**Pull request by dipeshfrollo**
[SDK-460] Remove tracking_type from create budget

[SDK-460]: https://frollo.atlassian.net/browse/SDK-460

### CODE REVIEW

**Review by sincychacko-frollo — APPROVED** [review]

File: Sources/Budgets/Budgets.swift [hunk 1]
```diff
@@ -256,11 +256,10 @@ public class Budgets: CachedObjects, ResponseHandler {
                                     periodAmount: Decimal,
                                     imageURL: String? = nil,
                                     startDate: String? = nil,
-                                    trackingType: Budget.TrackingType,
```
Comment (nickfrollo) [inline Sources/Budgets/Budgets.swift:259]: Don't forget to remove the parameter from the documentation :)

**Review by nickfrollo — CHANGES_REQUESTED** [review]
Just check on the docs in comments please

File: Sources/Budgets/Budgets.swift (hunk 1 above)
Comment (dipeshfrollo) [reply Sources/Budgets/Budgets.swift:259]: Updated. Thanks 😊

**Review by nickfrollo — APPROVED** [review]
Thanks!

**Pull request merged**